s3/src/main/scala/akka/stream/alpakka/s3/auth/Utils.scala

```diff
@@ -0,0 +1,15 @@
+package akka.stream.alpakka.s3.auth
+
+import javax.xml.bind.DatatypeConverter
+
+import akka.util.ByteString
+
+object Utils {
```

Comment:
Utils-classes are an anti-pattern, right? Consider moving these methods into a package object if they are needed in many places.